### PR TITLE
chore: rebase and fix literal compilation

### PR DESCRIPTION
**Pull request by cpcloud**
Fixes literal compilation for temporal types in the MS SQL backend.

### CODE REVIEW

**Review by gforsyth — APPROVED** [review]
Looks good!

File: poetry-overrides.nix [hunk 1]
```diff
@@ -18,9 +18,9 @@ self: super: {
       })
     ];
   });
-  pyodbc = (super.pyodbc.override { preferWheel = false; }).overridePythonAttrs (attrs: {
-    nativeBuildInputs = attrs.nativeBuildInputs or [ ] ++ [
-      self.pkgs.unixODBC
-    ];
+  pyodbc = super.pyodbc.overridePythonAttrs (attrs: {
+    preFixup = attrs.preFixup or "" + ''
```
[the remark this answers is not in the document]
Comment (cpcloud) [reply poetry-overrides.nix:22]: Yeah ... 😒

**Comment by gforsyth** [comment]
Ack, this changed during review. I'm on mobile and might need a bigger screen for this one

**Comment by cpcloud** [comment]
Yeah, sorry about that 😬

**Review by kszucs — APPROVED** [review]

**Pull request merged**